firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/DisplayCallbacksImpl.java

```diff
@@ -72,6 +72,11 @@ public class DisplayCallbacksImpl implements FirebaseInAppMessagingDisplayCallba
     wasImpressed = false;
   }
 
+  @VisibleForTesting
+  boolean hasImpressed() {
+    return wasImpressed;
```

Comment:
curious why the method is called `hasImpressed`, while it accesses the variable called `wasImpressed`.

Reply:
Changed the method name to wasImpressed. This is introduced just for writing unit test.